src/tools/genpolicy/rules.rego

```diff
@@ -38,8 +38,7 @@ default StartTracingRequest := false
 default StatsContainerRequest := true
 default StopTracingRequest := false
 default TtyWinResizeRequest := true
-default UpdateContainerRequest := false
```

Comment:
UpdateContainerRequest seems unrelated to this PR - let's keep it here.

Reply:
Good catch! updated